tracer/src/Datadog.Trace/SamplingPriorityValues.cs

```diff
@@ -32,5 +37,23 @@ internal static class SamplingPriorityValues
         /// code or configuration (e.g. the rules sampler).
         /// </summary>
         public const int UserKeep = 2;
+
+        [return: NotNullIfNotNull("samplingPriority")]
+        internal static string? ToString(int? samplingPriority)
+        {
+            return samplingPriority switch
+            {
+                2 => "2",
+                1 => "1",
+                0 => "0",
+                -1 => "-1",
+                null => null,
+                not null => samplingPriority.Value.ToString(CultureInfo.InvariantCulture)
+            };
+        }
+
+        internal static bool IsKeep(int? samplingPriority) => samplingPriority > 0;
```

Comment:
I thought you said never to use `>` and `<` for sampling 😛

Reply:
Yeah, I thought about it and realized I was focusing on the wrong thing when I said that.

You shouldn't compare the different `KEEP` values to each other or the different `DROP` values to each other. Despite the name, there are no "priorities." For example, the fact that `MANUAL_KEEP` > `AUTO_KEEP` has no meaning. 

However, it should be safe to assume that any value `> 0` is a `KEEP`, and any value `<= 0` is a `DROP`. We will never add a positive value that means `DROP` or a negative value that means `KEEP`. Unless we're horrible, evil people.

Reply:
> Unless we're horrible, evil people.

:stare: :awkward-monkey: